Library/Homebrew/utils/livecheck.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "open3"
+
+module Livecheck
+  def livecheck_formula_response(formula_name)
+    ohai "- livecheck formula : #{formula_name}"
+    command_args = [
+      "brew",
+      "livecheck",
+      formula_name,
+      "--quiet",
+    ]
+
+    response = Open3.capture2e(*command_args)
+    parse_livecheck_response(response)
+  end
+
+  def parse_livecheck_response(response)
+    output = response.first.delete(" ").split(/:|==>|\n/)
```

Comment:
Why `.first`?

Reply:
This is because when we were using `Open3.capture2e`, the response was a 2 element array, with the second element being the process status.  

Since we are using `Utils.popen_read` now, I'l remove it.